Source/JavaScriptCore/llint/WebAssembly.asm

```diff
@@ -604,17 +604,29 @@ op(js_to_wasm_wrapper_entry, macro ()
 
     const AccumulatorTag = invalidGPR # Only used for JSVALUE32_64
```

Comment:
It's weird that the option for this is called IPInt but this code is in the LLInt implementation.

Reply:
Changed